src/realm_dart.cpp

```diff
@@ -94,4 +94,24 @@ RLM_API void realm_dart_invoke_unlock_callback(bool success, void* unlockFunc)
 }
 
 // stamped into the library by the build system (see prepare-release.yml)
-RLM_API const char* realm_dart_library_version() { return "0.8.0+rc"; }
+RLM_API const char* realm_dart_library_version() { 
+    return "0.8.0+rc"; 
+}
```

Comment:
This will break the prepare-release workflow - it does simple string replacement without parsing the syntax tree.

Reply:
thanks for noticing will revert and put a comment there